### ISSUE TITLE
Can't build testsys tools image due to `go mod vender` failure for `aws-iam-authenticator`

### ISSUE THREAD

**Issue by etungsten** [bottlerocket-os/bottlerocket-test-system#804]
`make tools` fails due to 
```
#43 [kubernetes-build 5/8] RUN go mod vendor
#43 sha256:5e03c3442e5be6e1024edc36039e4e9baf4e9eb71165ab22d30b127d705700f2
#43 139.9 go: github.com/storageos/go-api@v2.2.0+incompatible: reading github.com/storageos/go-api/go.mod at revision v2.2.0: git ls-remote -q origin in /home/builder/go/pkg/mod/cache/vcs/f4474392d641b836e8296e236f2861d83ba42188d7ff117e30acb6b06774a32d: exit status 128:
#43 139.9 	fatal: could not read Username for 'https://github.com': terminal prompts disabled
#43 139.9 Confirm the import path was entered correctly.
#43 139.9 If this is a private repository, see https://golang.org/doc/faq#git_https for additional information.
#43 ERROR: executor failed running [/bin/sh -c go mod vendor]: exit code: 1
```

which is the same issue described in https://github.com/bottlerocket-os/bottlerocket/pull/2956

Need to bump `aws-iam-authenticator` to 0.6.8, the latest version packaged in Bottlerocket

**Comment by jpmcb** [comment]
FYI - https://github.com/bottlerocket-os/bottlerocket-test-system/pull/805

**Comment by jpmcb** [comment]
FYI - https://github.com/bottlerocket-os/bottlerocket-test-system/pull/805

**Comment by ecpullen** [comment]
This was fixed by #810
